packages/cactus-plugin-ledger-connector-fabric/src/test/typescript/fixtures/go/lock-asset/chaincode-typescript/package.json

```diff
@@ -0,0 +1,62 @@
+{
```

Comment:
@jscode017 Question for this entire directory: Is it strictly necessary for this to be under version control? Can't we re-use one of the existing example contracts in the FAIO image and reduce the diff of the PR?

Reply:
@petermetz do you mean share the same package.json with other contracts?

Reply:
resolved